EATSSU/App/Sources/Data/Firebase/LaunchSourceManager.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  LaunchSourceManager.swift
+//  EATSSU
+//
+//  Created by 황상환 on 3/23/25.
+//
+
+import Foundation
+import UIKit
+import FirebaseAnalytics
+
+enum LaunchSource: String {
+    case icon = "icon"
+    case localNotification = "local_notification"
+    case widget = "widget"
+    case background = "background"
+    
+}
+
+class LaunchSourceManager {
```

Comment:
```suggestion
final class LaunchSourceManager {
```
해당 클래스는 싱글톤패턴이니 서브클래싱 되지 않겠네요!
final 붙여 클래스 선언해도 좋을 것 같습니다.